### DISCUSSION TITLE
Shifting width on large screens

### DISCUSSION THREAD

**Discussion by jcrd** [thegeeklab/hugo-geekdoc#233] [Q&A] [answered]
![shift](https://user-images.githubusercontent.com/44040597/140651321-c27cdd73-189a-4e61-a1f6-cbb52e1464dc.gif)

As seen in the above gif, the entire site's width shifts when navigating between links.
This seems to depend on whether the footer is in view on a given page without vertical scrolling.

This was tested on a `1920x1080` screen in both:
* chromium 95.0.4638.54
* firefox 93.0

**Comment by xoxys** [comment, marked as answer]
Not sure if I can follow. It is normal that the content shifts if you switch from a page that fits into the screen (without a vertical scrollbar) and a page that needs a vertical scrollbar.

As the scrollbar itself requires space, the rest of the page need to shifted.

**Comment by jcrd** [comment]
Oops, good point. That explains why there's no issue in `qutebrowser` which doesn't display a scrollbar by default.
Thanks for the prompt followup.
